src/ViewRendererBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ChameleonSystem\ViewRendererBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder();
+        $root = $treeBuilder->root('chameleon_system_view_renderer');
+        $root->addDefaultsIfNotSet();
+
+        $root
+            ->children()
+                ->scalarNode('css_dir')
```

Comment:
The name of this option could be more expressive. Perhaps even `css_output_dir` is enough?

Please also add a short description of the option using the `info` method.

Reply:
info added

The name is unchanged: Its not only an output dir but also a working dir.
(So output, working, storage, ... is not correct)